.gitignore

```diff
@@ -54,4 +54,5 @@ Vagrantfile
 .swp
 *.lock
 .buildpath
+.idea
```

Comment:
https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Would you like me to remove this? I saw you had quite a few IDE specific ignores but not PHPStorm so I added it but I can remove

Reply:
Yeah, I'll catch this and a few other things in the rebase we talked about in Slack just now :smile: